docs/source/release-notes/2021-1-3-Eos.md

```diff
@@ -0,0 +1,267 @@
+<li><a href="http://github.com/vircadia/Iamus/commit/12f635a0702f23e7014074b3ec5bed1e9035f553">view &bull;</a> Centralize date when places are not active in Places.dateWhenNotActive()</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/24020bdda186e1badcb0abce59417253b50b6626">view &bull;</a> Places: make functional queries for places: status=online,active</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/bafe73bb0cc025f407fdc85343cc8da379b6ca88">view &bull;</a> Add admin maint functions /api/maint/places/inactive and /api/maint/places/unhooked     with both GET and DELETE functions.</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/737e2d00cd09c428268660672460db12414258f8">view &bull;</a> Bump version to 2.4.3</li>
+</ul>
+
+#### Version 2.4.2
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/2521dc9e79fd1ceaabae68636bf659ce9b1659a9">view &bull;</a> Update ChangeLog.md</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/24e561e8701591b4f38d5c290cab095367b91abf">view &bull;</a> Update NPM package versions.     Gets rid of a possible security problem in one of the libraries.</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/6d064b757e8b68ba77d90b9ea0ff39481435e52c">view &bull;</a> Fix access permission bug where request would not complete if the a non-logged in user checked ADMIN access to an entity.</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/881162de6cd704ea7bf132db51f7d5f69b03d572">view &bull;</a> In returned Places info, return the domain's last heartbeat time     for current_last_update_time if there was not a current time set.</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/36359c2197035e4b0671e317d6df02a2e4c6c9a3">view &bull;</a> Add comments in api/v1/places about not needing to be logged in to fetch place info</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/2a42438b1b0d3f21c0a14016738ee4909723b7b0">view &bull;</a> Fix DELETE of places when using "DELETE /api/v1/user/places/:placeId". Also fix fetching info of single place using "GET /api/v1/user/places/:placeId". Closes issue #88</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/07816497bc136d4ebceb34249fd87369dfede9bb">view &bull;</a> Bump version to 2.4.2</li>
+</ul>
+
+#### Version 2.4.1
+
+<ul>
+<li><a href="http://github.com/vircadia/Iamus/commit/a269b7015d5993b0f84a7a2c1bfa11c4036434fc">view &bull;</a> Update ChangeLog.md</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/2982560c3749eacf6b29d4210052ff9bf220bd79">view &bull;</a> Remove comment and README references to ActivityPub since that is not     an explicit goal. Some federation system will be used but one has     not been chosen.</li>
+<li><a href="http://github.com/vircadia/Iamus/commit/03b96cb03447c774328fd927265025dec2a9172b">view &bull;</a> Bump version to 2.4.1</li>
```

Comment:
```suggestion
<li><a href="http://github.com/vircadia/Iamus/commit/a269b7015d5993b0f84a7a2c1bfa11c4036434fc">view &bull;</a> Update ChangeLog.md.</li>
<li><a href="http://github.com/vircadia/Iamus/commit/2982560c3749eacf6b29d4210052ff9bf220bd79">view &bull;</a> Remove comment and README references to ActivityPub since that is not     an explicit goal. Some federation system will be used but one has     not been chosen.</li>
<li><a href="http://github.com/vircadia/Iamus/commit/03b96cb03447c774328fd927265025dec2a9172b">view &bull;</a> Bump version to 2.4.1.</li>
```